resources/lang/en/forum.php

```diff
@@ -86,6 +86,15 @@
             'preview_hide' => 'Write',
             'submit' => 'Post',
 
+            'necropost' => [
+                '_' => 'This topic has been inactive for a while. Only post here if you have a specific reason to do so.',
```

Comment:
This key isn't being used together with a related key in the same translation, so name it  something besides `_`